ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -52,6 +52,8 @@ index.search.slowlog.threshold.query.info: 5s
 index.search.slowlog.threshold.query.debug: 2s
 index.search.slowlog.threshold.query.trace: 500ms
 
+index.refresh_interval: 5s
```

Comment:
I don't know that we want this here. This means that it will be for all indexes. We may not want that on users/groups/apps.

Also I think this will get overwritten on each deploy. https://github.com/dimagi/commcare-hq/blob/8667a4159ad39d3307f92a3812d74006d610fd1b/corehq/ex-submodules/pillowtop/es_utils.py#L20